package.json

```
@@ -5,7 +5,8 @@
   "main": "src/index.js",
   "scripts": {
     "build": "tsc",
-    "test": "jest",
+    "test": "jest -c ./jest.config.unit.js",
+    "integration": "jest -c ./jest.config.integration.js",
```

```suggestion
    "integration-test": "jest -c ./jest.config.integration.js",
```

What do you think about the above two suggestions?